src/main/java/step3/CarRacing.java

```diff
@@ -0,0 +1,52 @@
+package step3;
+
+import java.util.Random;
+
+public class CarRacing {
+	private static final int CAN_MOVE_NUM = 4;
+	private static final int RANDOM_LIMIT = 9;
+	private static final String MOVE_MARK = "-";
+	private static final Random random = new Random();
+
+	public static void main(String[] args) {
+		int carCount = InputView.requestHowManyCar();
+		int times = InputView.requestHowManyTimes();
+
+		StringBuilder[] cars = getNewCars(carCount);
+		StringBuilder[] raceResult = new StringBuilder[times];
+
+		for (int i = 0; i < times; i++) {
+			raceResult[i] = moveForward(cars);
+		}
+
+		ResultView.printResult(raceResult);
+	}
+
+	public static StringBuilder[] getNewCars(int carCount) {
+		StringBuilder[] cars = new StringBuilder[carCount];
+		for (int i = 0; i < cars.length; i++) {
+			cars[i] = new StringBuilder();
+		}
+		return cars;
+	}
+
+	private static StringBuilder moveForward(StringBuilder[] cars) {
+		StringBuilder roundResult = new StringBuilder();
+		for (int i=0; i < cars.length; i++) {
+			cars[i] = canMoveForward(cars[i]);
+			roundResult.append(cars[i]).append("\n");
```

Comment:
OS 마다 줄바꿈을 인식하는게 다르기 때문에 jdk 1.7부터 도입된 현재 OS에 맞는 줄바꿈을 얻어오도록 하면 어떨까요?
```suggestion
			roundResult.append(cars[i]).append(System.lineSeparator());
```